packages/braintree-integration/src/braintree.ts

```diff
@@ -459,6 +460,38 @@ export interface BraintreeVenmoCreatorConfig extends BraintreeModuleCreatorConfi
 //     createInitOptions(options: Partial<VisaCheckoutInitOptions>): VisaCheckoutInitOptions;
 // }
 
+/**
+ *
+ * Braintree US Bank Account
+ *
+ */
+
+export interface UsBankAccountSuccessPayload {
+    accountNumber: string;
+    routingNumber: string;
+    ownershipType: string;
+    accountType: string;
+    firstName?: string;
+    lastName?: string;
+    businessName?: string;
+    billingAddress: {
+        streetAddress: string;
+        extendedAddress: string;
+        locality: string;
+        region: string;
+        postalCode: string;
+    };
+}
+
+export type BraintreeUsBankAccountCreator = BraintreeModuleCreator<BraintreeUsBankAccount>;
+
+export interface BraintreeUsBankAccount extends BraintreeModule {
+    tokenize(payload: {
+        bankDetails: UsBankAccountSuccessPayload;
+        mandateText: string;
+    }): Promise<{ nonce: string; details: Record<string, string> }>;
```

Comment:
May I ask you another TS interface for payload?

Reply:
Done